### PR TITLE
Prepare App for further usage

### PR DESCRIPTION
**Pull request by nupplaphil**
Some minor improvements.
- constructor contains just variable settings
- added all "load" logic into the new `load()` method (renamed from `reload()`)
- removed static dependencies

=> Step by step PRs instead BIG PRs ;-)

### CODE REVIEW

File: src/App.php [hunk 1]
```diff
@@ -54,7 +53,7 @@ class App
 	public $contacts;
 	public $page_contact;
 	public $content;
-	public $data = [];
+	public $data         = [];
```
Comment (MrPetovan) [inline src/App.php]: ?

Comment (nupplaphil) [reply src/App.php]: IDE automation -> revert it ;-)

**Comment by nupplaphil** [comment]
OK, now I'm finished :X .. I updated this branch on https://friendica.philipp.info

**Comment by nupplaphil** [comment]
Fixing https://github.com/friendica/friendica/pull/7508#issuecomment-521679624 (`App->module` is usable again)

**Pull request merged**